app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -28,6 +29,27 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     has_many :middleware_messagings, :foreign_key => :ems_id, :dependent => :destroy
     has_many :middleware_server_groups, :through => :middleware_domains
 
+    standalone_operation :shutdown, 'Shutdown'
```

Comment:
Shouldn't this one (and some others) have `{:timeout => 0}` as default parameters?

Reply:
I believe the hawkular agent defaults this already, so no need to pass.